build/Codegen.cmake

```diff
@@ -79,7 +79,7 @@ function(generate_bindings_for_kernels)
   execute_process(
     COMMAND
       "${PYTHON_EXECUTABLE}" -c
-      "from distutils.sysconfig import get_python_lib;print(get_python_lib())"
+      "import torch; from pathlib import Path; print(Path(torch.__file__).parent.parent.absolute())"
```

Comment:
please don't undo this optimization without a good reason. why doesn't it work as written? can we split this change out?